### ISSUE TITLE
default messages are not found

### ISSUE THREAD

**Issue by greuze** [Microsoft/BotBuilder#1579]
My bot has default locale `en-us`, and it loads:

```
bot/locale/en-us/index.json
bot/node_modules/botbuilder/lib/locale/en/BotBuilder.json
```

When I sent a `Promt.confirm`, I get `confirm_yes` and `confirm_no` texts in buttons, even tough they are in `BotBuilder.json`. I tried changing locale to `en` with same results.

When I Manually call `session.gettext('en', 'confirm_no')` I get `null`, I must call `session.gettext('en', 'BotBuilder:confirm_no')` to get expected `no`, but that `BotBuilder` preffix is not added automatically by confirm promt.

**Comment by iassal** [comment]
This should be fixed in v3.4.4. This was addressed in https://github.com/Microsoft/BotBuilder/issues/1559 

Upgdate the package and retry.

**Comment by greuze** [comment]
Upgrading to 3.4.4 fixed the problem :dancer:
Thank you very much.
